lisp/plugins/list_layout/playing_widgets.py

```diff
@@ -148,19 +148,19 @@ def _update_timers(self, time):
 
     def _pause(self):
         self.cue.pause(
-            fade=AppConfig().get('listLayout.pauseCueFade', True))
+            fade=get_plugin('ListLayout').Config.get('pauseCueFade', True))
```

Comment:
We can get the configuration once in the `__init__`, and then `self.config.get(...)`.
Ideally we should get the configuration as an `__init__` arguments.

Reply:
Done; and - although I'm not sure it's necessarily an improvement - done.

Reply:
Thanks.
It's should keep thing simpler, in theory (see dependency-injection)